### ISSUE TITLE
Adding MLP module using R torch

### ISSUE THREAD

**Issue by chandryou** [OHDSI/DeepPatientLevelPrediction#1]
We need to add MLP module using R torch

**Comment by ted9219** [comment]
Hi Chan, just for clarifying, did you mean rTorch package ([this](https://cran.r-project.org/web/packages/rTorch/rTorch.pdf)) or torch package ([this](https://torch.mlverse.org/)) ? The rTorch needs a python environment with reticulate but the torch package doesn't. it seems that it is better to use the torch package if we can.

**Comment by chandryou** [comment]
@ted9219  Of course, I mean the latter (torch in R without dependency of python). :)
